HoloPipelines/pipelines/pipelineNifti2glb.py

```diff
@@ -0,0 +1,25 @@
+from components import compNifti2numpy
+from components import compNumpy2obj
+from components import compObj2glbWrapper
+import pathlib
+import sys
+
+
+def main(inputNiftiPath, outputGlbPath, threshold):
+    generatedNumpyList = compNifti2numpy.main(str(pathlib.Path(inputNiftiPath)))
+    generatedObjPath = compNumpy2obj.main(
+        generatedNumpyList,
+        threshold,
+        str(pathlib.Path.cwd().joinpath("output", "OBJ", "nifti2glb_tempObj.obj")),
+    )
+    generatedGlbPath = compObj2glbWrapper.main(
```

Comment:
Same here, unused variable. Make use of it.

Reply:
output path variable now used in final print of a pipeline